src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -662,18 +662,11 @@ private async Task<Commit> GetCommitAsync(string accountName, string projectName
                     HttpMethod.Get,
                     accountName,
                     projectName,
-                    $"_apis/git/repositories/{repoName}/commits",
+                    $"_apis/git/repositories/{repoName}/commits/{sha}?api-version=6.0",
```

Comment:
The version shouldn't be included in the request URI like this. This method takes an optional parameter for which version of the API to use.